.github/workflows/api.yml

```diff
@@ -76,4 +76,4 @@ jobs:
           cmd: |
             'version'
             'start-build reggie-api-master-build --follow -n 6e2f55-tools'
-            'tag -n ${NAMESPACE} ${IMAGE_NAME}:latest ${IMAGE_NAME}:${{env.DES_TAG}}'
+            'tag -n 6e2f55-tools reggie-api:latest reggie-api:${{env.DES_TAG}}'
```

Comment:
why not do the same as the web build with setting env vars? https://github.com/bcgov/rocketchat/blob/fix/kustomize-api-configmap/.github/workflows/web.yml#L85